### PR TITLE
Fix to get Flightcontrol working

### PR DESCRIPTION
**Pull request by flybayer**

### CODE REVIEW

File: flightcontrol/flightcontrol.json [hunk 1]
```diff
@@ -14,8 +14,10 @@
           "type": "fargate",
           "cpu": 0.25,
           "memory": 0.5,
-          "buildCommand": "yarn rw deploy flightcontrol",
-          "startCommand": "yarn rw deploy flightcontrol --serve",
+          "port": 8911,
+          "installCommand": "cd flightcontrol && yarn install --frozen-lockfile --production=false",
```
Comment (thedavidprice) [inline flightcontrol/flightcontrol.json:18]:
```suggestion
          "installCommand": "cd flightcontrol && NODE_ENV=development yarn install --immutable",
```

Ah, I see what's going on. Also, this is yarn 3 so no `production` option. How does the above look?

File: flightcontrol/flightcontrol.json [hunk 2]
```diff
@@ -14,8 +14,10 @@
           "type": "fargate",
           "cpu": 0.25,
           "memory": 0.5,
-          "buildCommand": "yarn rw deploy flightcontrol",
-          "startCommand": "yarn rw deploy flightcontrol --serve",
+          "port": 8911,
```
Comment (thedavidprice) [inline flightcontrol/flightcontrol.json:17]: We can set this port directly in the `rw deploy flightcontrol --serve` command, which I think it much better than having devs futz with getting it right here.

So if I update the command to serve at port 3000, then we can remove this, correct?

**Review by thedavidprice — CHANGES_REQUESTED** [review]
@flybayer see my questions and suggestions

File: flightcontrol/flightcontrol.json (hunk 1 above)
Comment (thedavidprice) [reply flightcontrol/flightcontrol.json:18]: Also, looks like we need to do the same on L29, correct?
```
"buildCommand": "yarn rw deploy flightcontrol web",
```

File: flightcontrol/flightcontrol.json (hunk 2 above)
Comment (flybayer) [reply flightcontrol/flightcontrol.json:17]: @thedavidprice correct

File: flightcontrol/flightcontrol.json (hunk 1 above)
Comment (flybayer) [reply flightcontrol/flightcontrol.json:18]: yes on both accounts

**Review by thedavidprice — APPROVED** [review]

**Pull request merged**